gui/screens/settings.py

```diff
@@ -479,6 +481,21 @@ def check_acknowledgement_checkbox(self, value: bool):
         self._network_acknowledgment_checkbox.set(value)
         return self
 
+    @allure.step('Verify the text for consent when changing RPC urls')
```

Comment:
Maybe it's better to name this step and also two next steps 'Get the text...'? Because here we are not verifying/checking yet, just getting text (verifying in test itself)

Reply:
sure, will fix.